### PR TITLE
feat: update fee warning to v1.1 styles

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/ui/CheckButton/index.jsx [hunk 1]
```diff
@@ -13,12 +13,12 @@ const CheckButton = ({children, active, onClick, size, align, isCheckBox}) => (
     large: size === 'l',
     small: size === 's'
   })} size="lg" color="link" block active={active} onClick={onClick}>
-    {children}
     {!isCheckBox && <FontAwesomeIcon className={classnames({"float-right": align === "right", "float-left": align !== "right", "text-primary": active, "text-secondary": !active})}
                      icon={active ? faCircleReg : faCircle}/>}
     {isCheckBox && <span className={classnames("d-inline-block check-box text-center", {'bg-secondary': !active, 'bg-primary': active, "float-right": align === "right", "float-left": align !== "right"})}>
       {active && <FontAwesomeIcon className="text-white" icon={faCheck}/>}
     </span>}
+    {children}
```
Comment (jrainville) [inline src/js/ui/CheckButton/index.jsx]: Is the text now to the right?

File: src/js/wizards/Sell/6_Margin/components/MarginSelectorForm.jsx [hunk 2]
```diff
@@ -68,14 +70,14 @@ class MarginSelectorForm extends Component {
...
 
-        {(feeMilliPercent || '0') !== '0' && <Fragment>
-          <h3 className="mt-4">{t('marginSelectorForm.ourFee')}</h3>
-          <div className="border rounded p-3">
-            {feeMilliPercent / 1000} %
-          </div>
-        </Fragment>}
+        {(feeMilliPercent || '0') !== '0' && <div className="clearfix mt-5">
+          <span className="float-left mr-3">
+          < RoundedIcon image={infoIcon} bgColor="blue" />
```
Comment (jrainville) [inline src/js/wizards/Sell/6_Margin/components/MarginSelectorForm.jsx]: You have a space between `<` and RoundedIcon

Comment (jrainville) [reply src/js/wizards/Sell/6_Margin/components/MarginSelectorForm.jsx]: Also, might miss a `tab`

File: src/js/ui/CheckButton/index.jsx (hunk 1 above)
Comment (richard-ramos) [reply src/js/ui/CheckButton/index.jsx]: The checkbox/radio have a float that will position it before the text.

**Pull request merged**